src/backend/distributed/sql/udfs/run_command_on_all_nodes/11.0-1.sql

```diff
@@ -0,0 +1,21 @@
+DROP FUNCTION IF EXISTS pg_catalog.run_command_on_all_nodes;
+
+CREATE FUNCTION pg_catalog.run_command_on_all_nodes(command text, parallel bool default true, OUT nodename text, OUT nodeport int, OUT success bool, OUT result text)
+	RETURNS SETOF record
+	LANGUAGE plpgsql
+	AS $function$
+DECLARE
+	workers text[];
+	ports int[];
+	commands text[];
+BEGIN
+    WITH citus_nodes AS (
+        SELECT * FROM pg_dist_node ORDER BY nodename, nodeport
```

Comment:
My comment somehow doesn't show up. We should filter with nodecluster  and maybe even `isActive`